resources/assets/js/helpers/router.ts

```diff
@@ -37,17 +37,21 @@ export const useUrlHash = (): void => {
 };
 
 export const useRouter = (
-  routes: Routes<any, React.ReactElement>,
+  routes: Routes<any, any>,
```

Comment:
You can easily define the type as `{title: string; component: React.ReactElement}`